README.rst

```diff
@@ -25,29 +25,19 @@ Currently the following checks are available:
 Running
 -------
 
-You can run the checks locally with
-`Taskotron <https://fedoraproject.org/wiki/Taskotron>`__. First,
-install it (you can
-follow the
-`Quickstart <https://qa.fedoraproject.org/docs/libtaskotron/latest/quickstart.html>`__).
-You'll also need the ``rpm``, ``dnf`` and ``libarchive-c`` Python 2 modules
-(``python2-rpm``, ``python2-dnf``, ``python2-libarchive-c``).
-Note that Taskotron unfortunately runs on Python 2, but the code in
-this repository is Python 3 compatible as well.
-
-Once everything is installed you can run the task on a Koji build
-using the
-``name-(epoch:)version-release`` (``nevr``) identifier.
+To run this task locally, execute the following command as root (don't do this
+on a production machine!)::
 
-.. code:: console
+  $ ansible-playbook tests.yml -e taskotron_item=<nevr>
 
-    $ runtask -i <nevr> -t koji_build runtask.yml
+where ``nevr`` is a Koji build ``name-(epoch:)version-release`` identifier.
 
-For example:
+For example::
 
-.. code:: console
+  $ ansible-playbook tests.yml -e taskotron_item=python-gear-0.11.0-1.fc27
```

Comment:
There was a `--debug` switch to `runtask`. How do I turn on debug logging now?

Reply:
Currently, when you use `runtask --debug`, we do two things:
1. Switch taskotron runner to debug mode
2. Append `-vv` to `ansible-playbook` invocation.

Since the task no longer runs on the same machine and in the same process as the runner, our setting no longer applies to libtaskotron libraries being used inside the task. However, you can set up your own ansible variable and if it is present, you can switch your task to debug output (for libtaskotron libraries, simple use `logging.getLogger('libtaskotron').setLevel(logging.DEBUG)`). Even better, we can expose an ansible variable taskotron_debug: bool` and you can react on that. Which means the task will show debug output even when run through `runtask` and there will be a standard way to toggle this for all tasks (that support that).

That being said, all of this might be of limited use and I'm actually not sure it's needed. Since the task is now run through ansible, you now longer see its standard output (at least not in realtime). Instead, you're required to create `{{artifacts}}/test.log` file containing log output of your task. That can be always put to debug level, if you want. Or you can create `test.log` with standard level output, and `test.log.debug` with debug level output, every time. It's your preference. We will always archive whole `{{artifacts}}` dir. Still, if you prefer a boolean toggle, I can expose it easily.

Reply:
keep it how it is, thanks for explanation